### ISSUE TITLE
vscode desktop file cant be launched by default

### ISSUE THREAD

**Issue by spacey-sooty** [wpilibsuite/WPILibInstaller-Avalonia#461]

**Comment by sciencewhiz** [comment]
Did you follow the directions at https://docs.wpilib.org/en/latest/docs/zero-to-robot/step-2/wpilib-setup.html

**Comment by spacey-sooty** [comment]
Yes. This is unrelated, it works through the launcher but not on the desktop. I get this error ![Image](https://github.com/user-attachments/assets/234126da-98dd-4dad-8487-3af139868994)

**Comment by sciencewhiz** [comment]
What that message tells you to do is exactly what the docs tell you to do. Are you saying you did that and still get the error?

**Comment by spacey-sooty** [comment]
Oh, I missed that section of the docs as well 🤦‍ out of curiosity is there a way to make the tabs like this selected based on the browser user agent? ![Image](https://github.com/user-attachments/assets/9ac7708b-c7d6-445b-9a55-ba3b0161c1e9)
